statistics/views.py

```diff
@@ -194,6 +258,22 @@ def training_participant(request, rid):
         raise PermissionDenied()
     return render(request, 'statistics/templates/training_participant.html', context)
 
+def studentmaster_ongoing(request, rid):
+    
+    context = {}
+    try:
+        context['model_label'] = 'Workshop / Training'
+        context['model'] = TrainingRequest.objects.get(id=rid)
+        current_batch = TrainingRequest.objects.filter(id=rid)
```

Comment:
What's the difference here b/w `TrainingRequest.objects.get(id=rid)` and `TrainingRequest.objects.filter(id=rid)`?

The IDs must be unique so you'll always get a single object from both.